homu/server.py

```diff
@@ -147,10 +155,13 @@ def rollup():
 
 @post('/github')
 def github():
+    logger = g.logger.getChild('github')
+
     response.content_type = 'text/plain'
 
     payload = request.body.read()
     info = request.json
+    logger.debug('info: %s', utils.remove_url_keys_from_json(info))
```

Comment:
While this is a brilliant hack, I'm concerned that it will impact the performance (a little bit) even when the debugging is off. It is too sad that Python's logging isn't based on macro! How do you think of it? Would that be necessary to introduce some flag such as `DEBUG`?